contracts/src/registry/RegistryDatastore.sol

```diff
@@ -2,43 +2,47 @@
 pragma solidity >=0.8.13;
 
 import {IRegistryDatastore} from "./IRegistryDatastore.sol";
+import {NameUtils} from "../utils/NameUtils.sol";
 
 contract RegistryDatastore is IRegistryDatastore {
-    uint256 LABEL_HASH_MASK = 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffff00000000;
```

Comment:
I think we still need to mask out the lower bits in the datastore, because that provides a guarantee to anyone reading the datastore storage about how token IDs will be treated.

Reply:
It still masks it out - I've just abstracted the logic into `NameUtils.getCanonicalId()`